assets/src/components/searchForm.tsx

```diff
@@ -257,64 +343,71 @@ export const SearchForm = ({
           >
             <CircleXIcon />
           </button>
-          <button
-            type="submit"
-            title="Submit"
-            className="c-search-form__submit"
-            onClick={onSubmit}
-            // TODO(design): add error states instead of using `disabled`
-            disabled={!isValidSearchText(inputText)}
-          >
-            <SearchIcon />
-          </button>
+          {comboboxType == "map_search" && (
+            <button
+              type="submit"
+              title="Submit"
+              className="c-search-form__submit"
+              onClick={onSubmit}
+              // TODO(design): add error states instead of using `disabled`
+              disabled={!isValidSearchText(inputText)}
+            >
+              <SearchIcon />
+            </button>
+          )}
+          {comboboxType == "select" && (
+            <button
+              // TODO: rename this class (follow-up PR)
+              type="button"
+              className="c-search-form__submit"
+              onClick={() => setAutocompleteEnabled(true)}
```

Comment:
question: what happens here if the user has the option typed out exactly, and then clicks this button or tabs onto it to submit it?

Maybe you can use `onClick={onSubmit}`?

Reply:
Ahh! Yes, that was needed, as well as passing controllerRef to the GrouplessAutocomplete! Totally missed out on checking that piece of functionality.